plugins/misp/help.md

```diff
@@ -78,16 +94,26 @@ This action is used to export all events in XML format.
 
 |Name|Type|Default|Required|Description|Enum|Example|
 |----|----|-------|--------|-----------|----|-------|
-|encode_attachments|boolean|True|True|Encode attachments in export|None|None|
-|event_id|string|None|False|Specify single event to export|None|None|
-|from|string|None|False|From date E.g. 2015-02-15|None|None|
-|last|string|None|False|Events within x amount of time E.g. 5d|None|None|
-|tags|[]string|None|False|Array of tags to include in results|None|None|
-|to|string|None|False|To date E.g. 2015-02-17|None|None|
+|encode_attachments|boolean|True|True|Encode attachments in export|None|True|
+|event_id|string|None|False|Specify single event to export|None|1|
+|from|string|None|False|From date E.g. 2015-02-15|None|2015-02-15|
+|last|string|None|False|Events within x amount of time E.g. 5d|None|5d|
+|tags|[]string|None|False|Array of tags to include in results|None|["tag"]|
+|to|string|None|False|To date E.g. 2015-02-17|None|2015-02-17|
 
 Example input:
 
 ```
+{
+  "encode_attachments": true,
+  "event_id": "1",
+  "from": "2015-02-15",
```

Comment:
Is there any time period limit for the api?

Reply:
There is no limit